Tests/Microsoft.AppCenter.Test.Functional/Distribute/DistributeUpdateTest.cs

```diff
@@ -272,20 +265,19 @@ public async Task DisableAuthomaticCheckUpdateTest()
 
         private string GetReleaseJson(string version, string shortVersion, bool isMandatory)
         {
-            var schemeJson = "{{" +
-                "id: 42," +
-                "version: '{0}'," +
-                "short_version: '{1}'," +
-                "release_notes: 'Fix a critical bug, this text was entered in App Center portal.'," +
-                "release_notes_url: 'https://mock/'," +
-                "android_min_api_level: 19," +
-                "download_url: 'http://download.thinkbroadband.com/1GB.zip'," +
-                "size: 4242," +
-                "mandatory_update: {2}," +
-                "package_hashes: ['9f52199c986d9210842824df695900e1656180946212bd5e8978501a5b732e60']," +
-                "distribution_group_id: 'fd37a4b1-4937-45ef-97fb-b864154371f0'" +
-                "}}";
-            return string.Format(schemeJson, version, shortVersion, isMandatory);
+            return $"{{" +
```

Comment:
You can combine `$` and `@` together to get a multiline interpolated string literal